MAUI/TimePicker/getting-started.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: Getting started with .NET MAUI Time Picker control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Time Picker (SfTimePicker) control and its basic features.
```

Comment:
description: Learn about getting started with Syncfusion .NET MAUI Time Picker (SfTimePicker) control and its basic features.

Reply:
Content modified.